server/bleep/src/webserver/middleware.rs

```diff
@@ -16,34 +16,73 @@ use tracing::error;
 #[derive(Serialize, Clone)]
 pub enum User {
     Unknown,
-    Authenticated {
+    Desktop {
+        api_token: String,
         login: String,
+        #[serde(skip)]
+        crab: Arc<dyn Fn() -> anyhow::Result<octocrab::Octocrab> + Send + Sync>,
+    },
+    Cloud {
+        org_name: String,
         api_token: String,
+        login: String,
         #[serde(skip)]
         crab: Arc<dyn Fn() -> anyhow::Result<octocrab::Octocrab> + Send + Sync>,
     },
 }
 
 impl User {
-    pub(crate) fn login(&self) -> Option<&str> {
-        let User::Authenticated { login, .. } = self else {
+    pub fn login(&self) -> Option<&str> {
+        match self {
+            User::Desktop { login, .. } => Some(login),
+            User::Cloud { login, .. } => Some(login),
+            _ => None,
+        }
+    }
+
+    pub(crate) fn org_name(&self) -> Option<&str> {
+        let User::Cloud { org_name, .. } = self else {
             return None;
         };
 
-        Some(login)
+        Some(org_name.as_ref())
     }
 
     pub(crate) fn github(&self) -> Option<octocrab::Octocrab> {
-        let User::Authenticated { crab, .. } = self else {
-            return None;
+        let crab = match self {
+            User::Unknown => return None,
+            User::Desktop { crab, .. } => crab,
+            User::Cloud { crab, .. } => crab,
         };
 
         crab().ok()
     }
 
+    pub(crate) fn api_token(&self) -> Option<&str> {
```

Comment:
Calling this `access_token` would make the naming consistent across the backend.